Sources/Array+.swift

```diff
@@ -38,6 +38,27 @@ public extension Collection where Element: Hashable {
     }
 }
 
+public extension RangeReplacableCollection where Self: RandomAccessCollection {
+    /// Returns a collection with no duplicate elememts.
+	    ///
+	    /// NOTE: The elementa are not garanteed to be in the same order as the original colection.
+	    ///
+	    /// Complexity: O(n), where n is the number of elements in the collection.
+	    public func unified() -> Self {
+	        var set = Set<Element>()
+	        var r = Self()
```

Comment:
`r` is not used. Please refactor this code.

Reply:
I fixed it and removed the `extension Array` because it was redundant.